src/UserslimguAppDataLocalTemptmpsroyqvhk

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
Seems like there is a bad path somewhere because a temp file like this shouldn't be committed. Looking at the path, it looks like it was using hard coded path separators instead of `os.path.join`.

Reply:
Removed this file. It is accidentally checked in.